contrib/hyperscan/matching/input_matchers/test/config_test.cc

```diff
@@ -0,0 +1,148 @@
+#include <utility>
+#include <vector>
+
+#include "test/mocks/server/factory_context.h"
+
+#include "contrib/hyperscan/matching/input_matchers/source/config.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Matching {
+namespace InputMatchers {
+namespace Hyperscan {
+
+static const std::string yaml_string = R"EOF(
```

Comment:
no static non-pods -- you can lazy-init these using CONSTRUCT_ON_FIRST_USE or put them in your context.

Reply:
Thanks, fixed.